template.yml

```diff
@@ -1,6 +1,31 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Description: Sends log data from S3 to New Relic Logging.
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: NewRelic-log-ingestion-s3-RPB
+    Description: Send log data from a S3 bucket to New Relic Logging.
+    Author: New Relic
+    SpdxLicenseId: Apache-2.0
+    LicenseUrl: LICENSE
+    ReadmeUrl: README.md
+    Labels: ['newrelic', 'logs', 'logging', 'ingestion', 'lambda', 's3']
+    HomePageUrl: https://github.com/newrelic/aws_s3_log_ingestion_lambda
+    SemanticVersion: 1.1.10
```

Comment:
From where this SemanticVersion comes? I think it should be 1.1.3 (I just bumped it on your PR a few minutes ago)

Reply:
thanks. I was testing my own copy of the application - where I changed the application name. 
The versioning should follow the official cadence.